### PR TITLE
6 script for dynamic cards

### LINKED ISSUE #6: Make the script for generating a card from JS
Write the Javascript that will be used to generate a div for the keyword cards dynamically.

### LINKED ISSUE #9: Make the landingpage responsive
The page index.html should have a responsive design, to work well on different devices.

### LINKED ISSUE #12: Create the search function
We gotta make the search function on the page to search and filter through all the cards. The search function should preferably filter for each input by hiding elements that doesn't match, without having to use a search button.

### PR DESCRIPTION
**Pull request by OscarLarm**
This pull request adds 2 scripts, and some responsive design.
The first script is the outline for generating cards from arrays with a title and a description. The second script adds functionality to the search bar with a fuzzy search algorithm. 
Also adds media queries for resizing and rearranging elements based on the pixel width of the page to make the site more responsive and easier to use on different devices.

\
**Details**
- Creates a script for generating cards from arrays
- Creates a script for search functionality
- Add responsiveness to page based on page width with media queries
  - Mobile (<= 768px)
  - Tablet/Smaller monitor (769px <= x <= 1024px)
  - Desktop (>= 1025px)

\
**References**

Close #6 
Close #9 
Close #12

### CODE REVIEW

File: frontend/search.js
Comment (annk15) [inline frontend/search.js]: This is clever but I wonder if we should increase the delay to 500 (half a second) or even 1 second?
With half a second the person has to press a letter before the beat in this clip to prevent the search for occurring.
https://www.youtube.com/watch?v=XKkREJYOdTA

File: frontend/style.css
Comment (annk15) [inline frontend/style.css]: I think it will look better if this and the other comments in this file is kept on a single line

File: frontend/search.js
Comment (annk15) [inline frontend/search.js]: I don't understand this, can this comment be updated to better describe the purpose of this function?

File: frontend/generateCard.js
Comment (annk15) [inline frontend/generateCard.js]: This part is commented out so it can be removed

**Review by annk15 — CHANGES_REQUESTED** [review]
This looks super nice!
I found some minor things I think should be changed before we merge it.
If you don't agree you can just reply to the comment or push the new changes to the same branch and reply to the comment that it has been fixed, I will mark the comment as resolved and  approve the PR when everything is resolved.

**Comment by OscarLarm** [comment]
Fixed the issues

**Review by annk15 — APPROVED** [review]
Looks, good! 🚀

**Pull request merged**
